### PR TITLE
refactor(core): Rename runner broker uri

### PR DESCRIPTION
**Pull request by tomi**
## Summary

Rename `N8N_RUNNERS_N8N_URI` to `N8N_TASK_BROKER_URI` to match what launcher is using.

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/@n8n/task-runner/src/config/base-runner-config.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ class HealthcheckServerConfig {
 
 @Config
 export class BaseRunnerConfig {
-	@Env('N8N_RUNNERS_N8N_URI')
+	@Env('N8N_TASK_BROKER_URI')
```
Comment (ivov) [inline packages/@n8n/task-runner/src/config/base-runner-config.ts:17]: Should we rename also `n8nUri` for consistency?

Also the prefix is different now: `N8N_RUNNERS_` vs `N8N_TASK_BROKER`. Should we use the same everywhere?

Comment (tomi) [reply packages/@n8n/task-runner/src/config/base-runner-config.ts]: Good point! Updated the PRs to use `N8N_RUNNERS_` prefix for all

File: packages/@n8n/task-runner/src/js-task-runner/__tests__/js-task-runner.test.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ describe('JsTaskRunner', () => {
 				...defaultConfig.baseRunnerConfig,
 				grantToken: 'grantToken',
 				maxConcurrency: 1,
-				n8nUri: 'localhost',
+				taskBrokerUri: 'localhost',
```
Comment (ivov) [inline packages/@n8n/task-runner/src/js-task-runner/__tests__/js-task-runner.test.ts:37]: Shouldn't this have scheme and port?

https://github.com/n8n-io/task-runner-launcher/blob/37515673b39f9b61e996a98002543b0c638c019f/internal/config/validate_url_test.go#L29-L35

File: packages/@n8n/task-runner/src/config/base-runner-config.ts [hunk 3]
```diff
@@ -14,8 +14,8 @@ class HealthcheckServerConfig {
 
 @Config
 export class BaseRunnerConfig {
-	@Env('N8N_RUNNERS_N8N_URI')
-	n8nUri: string = '127.0.0.1:5679';
+	@Env('N8N_RUNNERS_TASK_BROKER_URI')
+	taskBrokerUri: string = '127.0.0.1:5679';
```
Comment (ivov) [inline packages/@n8n/task-runner/src/config/base-runner-config.ts:18]: Shouldn't this have scheme?

https://github.com/n8n-io/task-runner-launcher/blob/37515673b39f9b61e996a98002543b0c638c019f/internal/config/validate_url_test.go#L29-L35

File: packages/@n8n/task-runner/src/task-runner.ts [hunk 4]
```diff
@@ -92,7 +92,7 @@ export abstract class TaskRunner extends EventEmitter {
 		this.maxConcurrency = opts.maxConcurrency;
 		this.idleTimeout = opts.idleTimeout;
 
-		const wsUrl = `ws://${opts.n8nUri}/runners/_ws?id=${this.id}`;
+		const wsUrl = `ws://${opts.taskBrokerUri}/runners/_ws?id=${this.id}`;
```
Comment (ivov) [inline packages/@n8n/task-runner/src/task-runner.ts:95]: Do we somehow have an incosistency if the runner expects a scheme-less and port-less URI but the launcher requires them?

**Review by ivov — COMMENTED** [review]
One question, it might just be confusion on my part.

File: packages/@n8n/task-runner/src/js-task-runner/__tests__/js-task-runner.test.ts (hunk 2 above)
Comment (tomi) [reply packages/@n8n/task-runner/src/js-task-runner/__tests__/js-task-runner.test.ts:37]: It uses it directly [here](https://github.com/n8n-io/n8n/blob/rename-runner-n8n-uri-env-var/packages/@n8n/task-runner/src/task-runner.ts#L95). So it's not a URI. We should either rename it or make it use the uri

**Comment by tomi** [comment]
Closed in favor of #12040

**Pull request closed without merging**